cpp/src/groupby/sort/aggregate.cpp

```diff
@@ -538,9 +538,9 @@ void aggregate_result_functor::operator()<aggregation::MERGE_M2>(aggregation con
  */
 auto column_view_with_common_nulls(column_view const& column_0, column_view const& column_1)
 {
-  rmm::device_buffer new_nullmask = cudf::bitmask_and(table_view{{column_0, column_1}});
-  auto null_count                 = cudf::count_unset_bits(
-    static_cast<cudf::bitmask_type const*>(new_nullmask.data()), 0, column_0.size());
+  auto table                      = table_view{{column_0, column_1}};
+  auto [new_nullmask, null_count] = cudf::bitmask_and(table);
```

Comment:
```suggestion
  auto [new_nullmask, null_count] = cudf::bitmask_and(table_view{{column_0, column_1}});
```
By giving this a name you are passing in an lvalue and can't use move semantics inside `bitmask_and` I don't see where this table is used anywhere else, so I would suggest you keep it as an rvalue here.